pyproject.toml

```diff
@@ -5,3 +5,12 @@ skip = ["tests/imports.pyi"]
 
 [tool.black]
 force-exclude = ".*\\.pyi"
+
+[tool.mypy]
+show_error_codes = true
+check_untyped_defs = true
+disallow_any_generics = true
+warn_redundant_casts = true
+warn_return_any = true
```

Comment:
Why enable this warning? I always disable it, because it doesn't make any sense to me: `Any` should be accepted anywhere.

Reply:
Enabling this warning means that if we have a function annotated as returning `None`, but mypy can't infer the return type (causing mypy to fallback to a return-type of `Any`), we get a mypy error. I find it useful, as it essentially acts as a flag for "Hey! Mypy can't understand this function! Maybe add some more type annotations to help mypy out a little bit?"